components/create_comment/create_comment.jsx

```diff
@@ -407,6 +416,8 @@ export default class CreateComment extends React.PureComponent {
         this.setState({
             showPostDeletedModal: false
         });
+
+        this.props.resetCreatePostRequest();
```

Comment:
Is this doing anything? In mattermost-redux that function is just returning an action object:

```
export function resetCreatePostRequest() {
    return {type: PostTypes.CREATE_POST_RESET_REQUEST};
}
```

Reply:
Oh nevermind, I see it